pkg/trace/obfuscate/sql_test.go

```diff
@@ -175,60 +175,88 @@ func TestSQLUTF8(t *testing.T) {
 
 func TestSQLTableFinder(t *testing.T) {
 	t.Run("on", func(t *testing.T) {
-		os.Setenv("DD_APM_FEATURES", "table_names")
+		os.Setenv("DD_APM_FEATURES", "table_names,normalize_sql_tables")
```

Comment:
Shouldn't we have tests were they are set individually? Currently we only have tests with them set together. What if setting just one will have an unexpected outcome? We wouldn't catch it now.

Reply:
Fair point, I broke out test cases so they are treated differently now, but the bulk of the cases runs with both features.